src/types.jl

```diff
@@ -34,6 +34,12 @@ object is stored in the `:dimensions` field. These fields can be accessed with
 `ustrip` and `dimension`, respectively. Many operators in `Base` are defined on
 `AbstractQuantity` objects, including `+, -, *, /, ^, sqrt, cbrt, abs`.
 
+"""
+    NoDims
+
+An empty struct indicating that something is not a quantity.
+"""
+struct NoDims end
```

Comment:
Um... you wrote the code inside a docstring of another struct... Maybe try moving this to the bottom?